packages/shared/src/components/cards/TrendingFlag.tsx

```diff
@@ -1,43 +1,11 @@
 import React, { ReactElement } from 'react';
-import classNames from 'classnames';
-import styles from './Card.module.css';
-import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+import { RaisedLabel, RaisedLabelType } from './RaisedLabel';
 
-export default function TrendingFlag({
-  trending,
-  listMode,
-}: {
-  trending: number;
-  listMode?: boolean;
-}): ReactElement {
-  const description = `${trending} devs read it last hour`;
+export function TrendingFlag({ trending }: { trending: number }): ReactElement {
```

Comment:
I think we must keep the `listMode` to support both list and card layouts.
```suggestion
interface TrendingFlagProps {
  trending: number;
  listMode?: boolean;
}

export function TrendingFlag({ trending, listMode }: TrendingFlagProps): ReactElement {
```